test_integration/test_plugin_static_policy.py

```diff
@@ -48,25 +48,24 @@
 import geopmpy.topo
 import geopmpy.pio
 
+
 def getSystemConfig():
-    try:
-        subprocess.check_call(shlex.split("geopmadmin"),
-                              stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    except subprocess.CalledProcessError:
-        print("geopmadmin check failed, there is an issue with the site configuration.\n")
-        raise
     settings = {}
     for option in ["--config-default", "--config-override"]:
         try:
             proc = subprocess.Popen(shlex.split("geopmadmin {}".format(option)),
                                     stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-            config_file = proc.stdout.readline()
-            settings.update(json.loads(open(config_file.strip(), "r").read()))
+            proc.wait()
```

Comment:
no other instance popen interaction in our tests call this wait.  what is the issue you're fixing here?

Reply:
There is a warning that shows up about unclosed files in python3.  This still needs to be fixed in lots of other places in the tests.